models/repo.go

```diff
@@ -198,6 +239,8 @@ type Repository struct {
 	CloseIssuesViaCommitInAnyBranch bool               `xorm:"NOT NULL DEFAULT false"`
 	Topics                          []string           `xorm:"TEXT JSON"`
 
+	TrustModel TrustModelType
```

Comment:
We need a migration since we add a new column

Reply:
done